packages/drivers/odsp-driver/src/odspSummaryUploadManager.ts

```diff
@@ -207,7 +209,9 @@ export class OdspSummaryUploadManager {
 
         return getWithRetryForTokenRefresh(async (options) => {
             const storageToken = await this.getStorageToken(options, "WriteSummaryTree");
-
+            if (storageToken === null) {
```

Comment:
Should we wrap the getStorageToken function to make it throw instead of having it to check everywhere?

Reply:
Yes, I thought of that but then I thought maybe every call does need to throw if token is null. But in that case caller may catch the error. So I think I will wrap it up in another function which will throw in case of null token.